src/mcp_gsuite/tools_gslide.py

```
@@ -6,7 +6,7 @@
     EmbeddedResource,
     LoggingLevel,
 )
-from . import gslides
+from . import gslide
```

Ruff (0.8.2)

`.gslide` imported but unused

Remove unused import: `.gslide`

(F401)